framework/console/controllers/QueueController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\console\controllers;
+
+use Yii;
+use yii\console\Controller;
+
+/**
+ * Runs the queue listener.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+class QueueController extends Controller
```

Comment:
@klimov-paul 

> there is no need for predefined controller for the queue processing

Since we talk about high-level approach in addition to low level approach ofcource we need such controller with default implementation. Ninjas who want non-PHP workers may write it even on Erlang if they want. Main idea is we should provide php workers from box for typical user.